include/pagmo/problems/luksan_vlcek1.hpp

```diff
@@ -0,0 +1,228 @@
+/* Copyright 2017 PaGMO development team
+
+This file is part of the PaGMO library.
+
+The PaGMO library is free software; you can redistribute it and/or modify
+it under the terms of either:
+
+  * the GNU Lesser General Public License as published by the Free
+    Software Foundation; either version 3 of the License, or (at your
+    option) any later version.
+
+or
+
+  * the GNU General Public License as published by the Free Software
+    Foundation; either version 3 of the License, or (at your option) any
+    later version.
+
+or both in parallel, as here.
+
+The PaGMO library is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received copies of the GNU General Public License and the
+GNU Lesser General Public License along with the PaGMO library.  If not,
+see https://www.gnu.org/licenses/. */
+
+#ifndef PAGMO_PROBLEM_LUKSANVLCECK_HPP
+#define PAGMO_PROBLEM_LUKSANVLCECK_HPP
+
+#include <cassert>
+#include <exception>
+#include <iostream>
+#include <stdexcept>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "../detail/constants.hpp"
+#include "../exceptions.hpp"
+#include "../problem.hpp" // needed for cereal registration macro
+#include "../types.hpp"
+
+namespace pagmo
+{
+
+/// Test problem from the Luksan and Vlcek
+/**
+ * Implementation of Example 5.1 in the report from Luksan and Vlcek.
+ *
+ * The problem is the Chanied Rosenbrock function with trigonometric-exponential
+ * constraints.
+ *
+ * Its formulation in pagmo can be written as:
+ *
+ * \f[
+ * \begin{array}{rl}
+ * \mbox{find:}      & -5 \le \mathbf x_i \le 5, \forall i=1..n\\
+ * \mbox{to minimize: } & \sum_{i=1}^{n-1}\left[100\left(x_i^2-x_{i+1}\right)^2 + \left(x_i-1\right)^2\right]\\
+ * \mbox{subject to:} & 3x_{k+1}^3+2x_{k+2}-5+\sin(x_{k+1}-x_{k+2}})\sin(x_{k+1}+x_{k+2}})
+ * +4x_{k+1}-x_k\exp(x_k-x_{k+1})-3 \le UB, \forall k=1..n-2 \\
+ *                    & 3x_{k+1}^3+2x_{k+2}-5+\sin(x_{k+1}-x_{k+2}})\sin(x_{k+1}+x_{k+2}})
+ * +4x_{k+1}-x_k\exp(x_k-x_{k+1})-3 \ge LB, \forall k=1..n-2 \\
+ * \end{array}
+ * \f]
+ *
+ * See: Lukšan, L., and Jan Vlcek. "Sparse and partially separable test problems for unconstrained and equality
```

Comment:
I would check if the "š" character survives intact in the doxygen -> sphinx transition. If it does, I'd use it consistently (elsewhere a normal "s" is used).